packages/documentation/styles/tables.scss

```diff
@@ -1,30 +1,26 @@
-@import '@3yourmind/kotti-ui/dist/variables.scss';
+@import '@3yourmind/kotti-ui/source/kotti-style/_variables.scss';
 
-table,
-.table-style {
+table:not(.kt-table) {
 	width: 100%;
 	hyphens: auto;
 	table-layout: auto;
 	border-collapse: collapse;
 
-	th,
-	.th-style {
+	th {
 		padding: var(--unit-2) var(--unit-1);
 		font-size: $font-size-sm;
 		line-height: 1em;
 		color: $lightgray-500;
 		text-align: inherit;
 		text-transform: uppercase;
-		border-bottom: $border-width solid $border-color;
```

Comment:
killed some of the variables and used their values instead until we can further refactor to use css vars everywhere